src/libs/workflows-app-utils.test.js

```diff
@@ -0,0 +1,92 @@
+import { resolveRunningCromwellAppUrl } from 'src/libs/workflows-app-utils';
```

Comment:
similar comment, recommend trying to write this file testing new code in `ts`, since Apps and Statuses have types which should help restrict the test code.